src/mmrelay/matrix/relay.py

```
@@ -0,0 +1,436 @@
+                        f'<a href="{reply_link}">In reply to</a> '
+                        f'<a href="{bot_link}">{bot_user_id}</a><br>'
+                        f"[{html.escape(original_sender_display)}]: {safe_original}"
+                    )
+                    content["formatted_body"] = (
+                        f"<mx-reply><blockquote>{blockquote_content}</blockquote></mx-reply>{formatted_body}"
+                    )
+                else:
+                    facade.logger.warning(
+                        f"Could not find original message for reply_to_event_id: {reply_to_event_id}"
+                    )
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)